modules/nw-dns-operator.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+// * networking/dns/dns-operator.adoc
+
+[id="dns-operator-{context}"]
+= DNS Operator
+
+The DNS Operator implements the `dns` API from the `operator.openshift.io` API
+group. The Operator manages a DNS Service for the cluster, and configures
+`kubelets` to tell individual containers to use the Service IP of the CoreDNS
```

Comment:
``` `kubelets` ``` looks strange, but the documentation is all over the place with how it refers to the kubelet: ``` the `kubelet` ```, `the kubelet`, `the Kubelet`, or just `Kubelet`.

Reply:
Or its more formal title, Sir Kubelet III, Esq.

Reply:
@Miciah I agree. Let me review upstream docs for a consistent way to reference the Kubelet.

Reply:
Although upstream docs are not 100% consistent, "kubelet" appears to be the most widely used. I will do the same.